src/map/hash_trie_map/mod.rs

```diff
@@ -856,5 +856,58 @@ impl<'a, K, V> Iterator for Iter<'a, K, V>
 
 impl<'a, K: Eq + Hash, V> ExactSizeIterator for Iter<'a, K, V> {}
 
+#[cfg(feature = "serde")]
+mod serde_impl {
```

Comment:
This need to be public.  Also I think it is nicer for the module to just be called "serde".

Reply:
Technically I think it doesn't need to be `pub` because trait impl visibility is strange ( https://users.rust-lang.org/t/visibility-of-trait-implementation/6789 ), but I agree that it makes more sense that way!  I've pushed these changes; pull request #4